src/test/provision/orbit/test_gdh.py

```diff
@@ -45,11 +45,18 @@ def setUp(self):
 
     def test_get_orbit(self):
         self.orbit_factory._orbit_from_child = MagicMock()
+        self.orbit_factory._describe_stack.return_value = {
+            'Outputs': [{
+                'OutputKey': 'SecurityGroup',
+                'OutputValue': 'sg-000001'
+            }]
+        }
 
         self.orbit_factory.get_orbit(self.orbit)
 
         self.cloudformation.describe_stack_resource.assert_called_once()
         self.orbit_factory._orbit_from_child.assert_called_once()
+        self.assertEqual(self.orbit._bastion_sgs[REGION], 'sg-000001')
```

Comment:
Nit: calling the "public" function `bastion_sg(REGION)` is better practice than accessing the "private" field directly.